internal/compiler/passes/lower_property_to_element.rs

```diff
@@ -73,18 +73,22 @@ pub(crate) fn lower_property_to_element(
                         ),
                     )
                 }
-            } else if has_property_binding(&child) {
-                let new_child = create_property_element(
-                    &child,
-                    property_name,
-                    extra_properties.clone(),
-                    default_value_for_extra_properties,
-                    element_name,
-                    type_register,
-                );
-                crate::object_tree::adjust_geometry_for_injected_parent(&new_child, &child);
-                new_child.borrow_mut().children.push(child);
-                child = new_child;
+            } else if child.borrow().repeated_as_embedding().is_some() {
+                todo!()
+            } else {
+                if has_property_binding(&child) {
```

Comment:
Maybe these should go into the same line. But I suggest to turn this into a match instead if/else if/else if.